.github/workflows/lint.yml

```diff
@@ -71,7 +71,7 @@ jobs:
       - uses: Swatinem/rust-cache@v2
         with:
           cache-on-failure: true
-      - run: cargo hack check
+      - run: cargo hack check --workspace --exclude op-reth
```

Comment:
think you will have to add a new job for op-reth, we still want to check lint for op code

Reply:
added a step to run `cargo check` on it with optimism feature